M2/BUILD/mahrud/package-review.sh

```diff
@@ -0,0 +1,26 @@
+owner=mahrud
+token=`cat ~/.github_token` # https://github.com/settings/tokens
+
+# List workflows:
+#curl -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/$owner/M2/actions/workflows
+
+if [[ -z "$1" ]];
+then package="FirstPackage"
+else package=$1
+fi
+
+shift
+
+if [[ -z "$1" ]];
+then ref="refs/master"
+else ref=$1
+fi
+
+echo package: $package
+echo     ref: $ref
+
+curl \
+  -u $owner:$token -X POST \
+  -H "Accept: application/vnd.github.v3+json" \
+  -d "{\"inputs\":{\"package\":\"$package\"}, \"ref\":\"$ref\"}" \
+  https://api.github.com/repos/$owner/M2/actions/workflows/1877698/dispatches
```

Comment:
What's the explanation of this URL?

Reply:
It's the API gateway for triggering a workflow dispatch event: https://docs.github.com/en/rest/reference/actions#create-a-workflow-dispatch-event

Reply:
Where did the number 1877698 come from?

Reply:
That's the workflow id. The commented out command a few lines above lists available workflow ids.

This script is for advanced use, hence why it's in this directory and not advertised. The web interface is the intended way to trigger this workflow.